ajax/libs/wangEditor/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "wangEditor",
+  "version": "3.0.0",
+  "filename": "wangEditor.min.js",
+  "description": "wangEditor, your simple open-source easy-use rich text editor!",
+  "homepage": "https://wangeditor.github.io/",
+  "author": {
+    "name": "wangfupeng1988",
+    "email": "wangfupeng1988@163.com",
+    "url": "https://github.com/wangfupeng1988"
+  },
+  "keywords": [
```

Comment:
So what keywords you suggest?
These keywords are good to google.
Upstream's keywords use Chinese, and we can't use it.